trunner/types.py

```diff
@@ -208,7 +208,7 @@ class TestOptions:
     target: Optional[str] = None
     bootloader: Optional[BootloaderOptions] = None
     shell: Optional[ShellOptions] = None
-    should_reboot: bool = True
+    reboot: bool = False
```

Comment:
I'm not sure if the `reboot` property means reboot BEFORE the test or AFTER the test (AFAIK right now we don't have tests which require rebooting after the test but maybe in the future we might?)

Please either name the property differently or at least add a comment

Reply:
I'd leave a comment to have consistent names for yaml properties.

Reply:
yeah, it was a more generic statement (wheter yaml property shouldn't be named `reboot_before` (and maybe `reboot_after` in the future if there is a need)